microsite/sidebars.json

```diff
@@ -358,6 +345,25 @@
         ]
       }
     ],
+    "Tooling": [
```

Comment:
What about?

```
Local development
├── Tools
    └──  CLI
├── Guides
    ├── Debugging
    └── Linking local packages
```

Reply:
The reasoning for renaming from "Local Development" was that it's not just local development, it's CI and production builds too